.yamllint

```diff
@@ -2,15 +2,19 @@
 extends: default
 
 rules:
-  line-length:
-    max: 80
-    level: warning
+  braces:
+    max-spaces-inside: 1
+    level: error
+  brackets:
+    max-spaces-inside: 1
+    level: error
   document-start: disable
-
+  line-length: disable
+  truthy: disable
```

Comment:
can we remove this? it is a good ansible practice to restrict boolean values to `true` and `false`

Reply:
Allowed values should also be False and True. By default truthy allows only ['true', 'false']. At this point, not sure if we define a custom list of allowed values or keep it disabled.